controllers/ovs/flowsconfig_reconciler.go

```diff
@@ -40,7 +40,7 @@ func NewFlowsConfigController(client reconcilers.ClientHelper,
 // Reconcile reconciles the status of the ovs-flows-config configmap with
 // the target FlowCollector ipfix section map
 func (c *FlowsConfigController) Reconcile(
-	ctx context.Context, target *flowsv1alpha1.FlowCollector) error {
+	ctx context.Context, target *flowsv1alpha1.FlowCollector, flpServiceName string) error {
```

Comment:
I think it's possible to avoid OVS reconciliation here, if we keep the same service regardless if it's flp or flp-ingester, it would avoid the OVS reconfig.
As you prefer, if you think it can make things simpler.

Reply:
Both deployment now use the same service, but now that the refactoring was done, I kept using an argument in the ovs reconciliation instead of relaying on a constant.

Is it fine for you?